src/main/java/org/broadinstitute/hellbender/tools/recalibration/covariates/ContextCovariate.java

```diff
@@ -91,9 +87,40 @@ public void recordValues(final SAMRecord read, final ReadCovariates values) {
         read.setReadBases(originalBases);
     }
 
+    /**
+     * Helper method: computes the correct offset to use in computations of covariate values.
+     * @param read the read
+     * @param lowQualTail every base quality lower than or equal to this in the tail of the read will be replaced with N.
```

Comment:
This method does not actually have a `lowQualTail` parameter as the javadoc claims -- add docs for the actual params.